src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ActivityExtensions.cs

```diff
@@ -26,6 +26,7 @@
 using Google.Protobuf.Collections;
 using OpenTelemetry.Configuration;
 using OpenTelemetry.Internal;
+using OpenTelemetry.Proto.Trace.V1;
 using OpenTelemetry.Trace;
 using OtlpCollector = OpenTelemetry.Proto.Collector.Trace.V1;
 using OtlpCommon = OpenTelemetry.Proto.Common.V1;
```

Comment:
You think we should just get rid of all the namespace aliases?

Reply:
Updated.

I had to keep `using OtlpTrace = OpenTelemetry.Proto.Trace.V1;` because `Status` is ambiguous but I was able to remove the others.